docs/azure/sdk/logging.md

```diff
@@ -213,16 +213,17 @@ Using the Azure Service Bus library as an example, complete the following steps:
 
 ## Log HTTP request and response bodies
 
+> [!NOTE]
+> This capability applies only to libraries using HTTP to communicate with an Azure service. Libraries based on alternative protocols, such as AMQP, don't support content logging. Unsupported examples include libraries for Azure services such as Event Hubs, Service Bus, and Web PubSub.
+
 When troubleshooting unexpected behavior with a client library, it's helpful to inspect the following items:
 
 - The HTTP request body sent to the underlying Azure service's REST API.
 - The HTTP response body received from the Azure service's REST API.
 
-By default, logging of the aforementioned content is disabled. Libraries for Azure services like Event Hubs, Service Bus, and Web PubSub don't support this type of logging. Those libraries are based on alternative protocols, such as AMQP.
-
-To enable logging of the HTTP request and response bodies, complete the following steps:
+By default, logging of the aforementioned content is disabled. To enable logging of the HTTP request and response bodies, complete the following steps:
 
-1. Set the client options object's <xref:Azure.Core.DiagnosticsOptions.IsLoggingContentEnabled%2A> property to `true`, and pass the options object to the client's constructor. For example:
+1. Set the client options object's <xref:Azure.Core.DiagnosticsOptions.IsLoggingContentEnabled%2A> property to `true`, and pass the options object to the client's constructor. For example, to log HTTP requests and responses for the Azure Key Vault Secrets library:
```

Comment:
It may be worth adding a note that the `DiagnosticOptions` can be found as a member of the client options passed to the HTTP service clients.